internal/highway/http/server.go

```diff
@@ -19,12 +19,14 @@ import (
 
 	"github.com/gin-gonic/gin"
 	"github.com/kataras/golog"
-
 	"github.com/sonr-io/sonr/internal/highway/x/ipfs"
+	"github.com/sonr-io/sonr/internal/highway/x/jwt"
 	"github.com/sonr-io/sonr/pkg/client"
 	"github.com/sonr-io/sonr/pkg/config"
 	hn "github.com/sonr-io/sonr/pkg/host"
 	ctv1 "github.com/sonr-io/sonr/x/channel/types"
+	swaggerfiles "github.com/swaggo/files"
+	ginSwagger "github.com/swaggo/gin-swagger"
```

Comment:
Maybe we could keep package names lowercase? I think that would help avoid confusion and also be a little bit more inline with Go's package/variable naming conventions.

Reply:
good point, i moved these imports without thinking about naming convention. ill make this change.